threeML/minimizer/minuit_minimizer.py

```diff
@@ -272,7 +226,7 @@ def _compute_covariance_matrix(self, best_fit_values):
 
         try:
 
-            covariance = np.array(self.minuit.matrix(correlation=False))
+            covariance = np.array(self.minuit.covariance)
```

Comment:
Is there a reason to cast to array here?

Reply:
Otherwise the type is `iminuit.util.Matrix`, so was trying to be consistent.